codegen-server/python/src/main/kotlin/software/amazon/smithy/rust/codegen/server/python/smithy/generators/PythonServerOperationHandlerGenerator.kt

```diff
@@ -69,11 +69,20 @@ class PythonServerOperationHandlerGenerator(
                     handler: #{SmithyPython}::PyHandler,
                 ) -> std::result::Result<$output, $error> {
                     // Async block used to run the handler and catch any Python error.
+                    let span = #{tracing}::span!(
+                        #{tracing}::Level::TRACE, "python",
+                        pid = #{tracing}::field::Empty,
+                        module = #{tracing}::field::Empty,
+                        filename = #{tracing}::field::Empty,
+                        lineno = #{tracing}::field::Empty
+                    );
+                    let guard = span.enter();
                     let result = if handler.is_coroutine {
                         #{PyCoroutine:W}
```

Comment:
What I did here is totally wrong and doesn't work as expected. The await point doesn't really bother from my tests, but the span needs to be created inside the tracing python wrapper to have the right info like filename, line number, etc..

Next commit will remove this from here and move it in another place where it will also resolve the holding the span over the await point.